frontend/pages/campaigns/[campaignId]/index.tsx

```diff
@@ -43,18 +44,62 @@ export default function CampaignDetail() {
 
   const numTotalCoupons = campaign.promisedAmount / campaign.couponDenomination;
   const numCouponsRedeemed = campaign.donations.primaryDonation.amount / campaign.couponDenomination;
+  const campaignStatus = getCampaignStatus(campaign.start, campaign.end);
+  const campaignIsUpcoming = campaignStatus === 'Upcoming';
+  const campaignHasEnded = campaignStatus === 'Completed';
 
-  const getPromisedAmountStatsCardChild = (): ReactNode => (
-    <Typography variant="h4">
-      was sponsored by {campaign.primaryDonor.name}. This will be donated to{' '}
-      <strong>
-        {campaign.charities.length} {campaign.charities.length === 1 ? 'charity' : 'charities'}
-      </strong>{' '}
-      when the campaign ends.
-    </Typography>
-  );
+  const getPromisedAmountStatsCardChild = (): ReactNode => {
+    if (campaignHasEnded) {
+      return (
+        <Typography variant="h4">
+          was sponsored by {campaign.primaryDonor.name}. This will be donated to{' '}
+          <strong>
+            {campaign.charities.length} {campaign.charities.length === 1 ? 'charity' : 'charities'}
+          </strong>
+          {campaign.charities.length === 1
+            ? '.'
+            : ' according to the redemption distribution shown under the charities section.'}
+        </Typography>
+      );
+    }
+
+    return (
+      <Typography variant="h4">
+        was sponsored by {campaign.primaryDonor.name}. This will be donated to{' '}
+        <strong>
+          {campaign.charities.length} {campaign.charities.length === 1 ? 'charity' : 'charities'}
+        </strong>{' '}
+        when the campaign ends.
+      </Typography>
+    );
+  };
 
   const getDonationAmountStatsCardChild = (): ReactNode => {
+    if (campaignIsUpcoming) {
+      return (
+        <Typography variant="h4">
+          was donated by the public. <strong>Be the first to donate when the campaign starts!</strong>
+        </Typography>
+      );
+    }
+
+    if (campaignHasEnded && campaign.donations.secondaryDonation.amount === 0) {
+      return (
+        <Typography variant="h4">
+          was donated by the public. Unfortunately, no personal contributions were made. However, the sponsored amount
```

Comment:
We didn't use the term personal contribution at all in this page so people may not know personal contribution = amount donated by public.
```suggestion
          was donated by the public. However, the sponsored amount
```